src/shared/layouts/FusionMainLayout.less

```diff
@@ -5,13 +5,18 @@
   height: 100%;
   min-height: 100vh !important;
 
+  &.wall {
+    .fusion-main-layout__content {
+      margin-top: 0;
+    }
+  }
```

Comment:
this one we can remove it so it does not appear again in merge conflicts
also it's better to remove the `classname` that it depends on it in the `MainLayout`